src/watcher.js

```diff
@@ -256,7 +256,7 @@ class Watcher extends SafeEventEmitter {
                         emitMessage($el);
                     }
 
-                    if ($el.children('.viewer-card').length) {
+                    if ($el.find('.viewer-card').length) {
```

Comment:
this feels sketch .. the reason i was using `children` is so we can check the direct descendants.. not up the tree as well

Reply:
It definitely is. Could limit the amount of traversal or 
`$el.children('.viewer-card').length || $el.children().children().children('.viewer-card').length`

`find()` will only traverse the descendants of element, but even then....

Reply:
Or a more of a `$el.children('.viewer-card').length || $el.find('div > div > .viewer-card').length`